proto/viam/service/navigation/v1/navigation.proto

```diff
@@ -59,6 +59,12 @@ service NavigationService {
     };
   }
 
+  rpc GetProperties(GetPropertiesRequest) returns (GetPropertiesResponse) {
```

Comment:
Add a comment over this for the API generated docs. Just a simple explanation of what it is.

Don't do this for the other rpcs, we'll add incrementally and i don't want to write anything wrong.

Reply:
Done!